CONTRIBUTING.md

```diff
@@ -0,0 +1,91 @@
+# Contributing
+
+## Contributing to the net35/main branch
+
+This branch is maintained for backwards compatibility with .NET Framework 3.5 and corresponds to the 6.x agent release version line.  It gets security patches and critical bug fixes, but no new features.  If you believe you have found a security flaw in this branch of the agent, please see the section about vulnerabilities in our top-level [README](./README.md).
+
+Contributions are always welcome. Before contributing please read the
+[code of conduct](./CODE_OF_CONDUCT.md) and [search the issue tracker](../../issues); your issue may have already been discussed or fixed in `main`. To contribute,
+[fork](https://help.github.com/articles/fork-a-repo/) this repository, commit your changes, and [send a Pull Request](https://help.github.com/articles/using-pull-requests/).
+
+Note that our [code of conduct](./CODE_OF_CONDUCT.md) applies to all platforms and venues related to this project; please follow it in all your interactions with the project and its participants.
+
+## Feature Requests
+
+**Note**: since this is a maintenance branch, it is unlikely that we will prioritize any feature requests on this branch. 
+
+Feature requests should be submitted in the [Issue tracker](../../issues), with a description of the expected behavior & use case, where they’ll remain closed until sufficient interest, [e.g. :+1: reactions](https://help.github.com/articles/about-discussions-in-issues-and-pull-requests/), has been [shown by the community](../../issues?q=label%3A%22votes+needed%22+sort%3Areactions-%2B1-desc).
+Before submitting an Issue, please search for similar ones in the
+[closed issues](../../issues?q=is%3Aissue+is%3Aclosed+label%3Aenhancement).
+
+## Pull Requests
+
+**Note**: in order to enable the correct CI to run on PRs into `net35/main`, add the `net35` tag to the PR after submitting it in GitHub.
+
+### Version Support
+
+When contributing, please keep in mind that New Relic customers (that's you!) are running many different versions of .NET, some of them pretty old. Changes that depend on the newest version of .NET Framework or Core will probably be rejected, especially if they replace something backwards compatible.  Code in `src/Agent` or `src/NewRelic.Core` needs to be compatible with .NET Framework 3.5.  Backwards compatibility is less important for code that lives in `tests`, but still should not gratuitously require features only available in the latest .NET Framework or Core.
```

Comment:
We didn't move NewRelic.Core into src, as in main. It remains in C:\git\newrelic-dotnet-agent\src\Agent\NewRelic\Agent.

Reply:
Good catch, thanks.